rpc/block_svc_routines.c

```diff
@@ -1043,7 +1045,11 @@ glusterBlockCreateRemoteAsync(blockServerDefPtr list,
   ret = 0;
   for (i = 0; i < mpath; i++) {
     if (args[i].exit) {
-      ret = -1;
+      if (args[i].exit == GB_BLOCK_NAME_EXIST) {
+        ret = GB_BLOCK_NAME_EXIST;
```

Comment:
should we break here ?

Reply:
For this case, my last commit have supported to clean all the 3 nodes(N1, N2 and N4), but this version, this case won't work, it won't even clean the N4.

I meant to support it in the next followed patches later after this.

Reply:
For this case there will be a lot of changes to adapt it.

Reply:
@lxbsz IMHO, without this the PR is incomplete.

Please take your time and add a new patch to the PR itself later.

I will hold this PR till then.

Thanks!

Reply:
sure.
Thanks.